src/app/components/PublishersTable/index.tsx

```diff
@@ -95,6 +96,11 @@ export default function PublishersTable({
                   </div>
                 )}
               </td>
+              <td className="px-6 py-6 whitespace-nowrap text-right">
+                {publisher.blocked && (
+                  <div className="ml-40">Site blocklisted!</div>
```

Comment:
We try to move i18n step by step, would be nice if you could move texts here already: https://github.com/getAlby/lightning-browser-extension/blob/master/src/i18n/locales/en.json

Reply:
sure, I can do that but I'll need some sample code to see how to pull the text from the i18n file and show to user.

Reply:
Does this help?  
https://github.com/MiguelMedeiros/lightning-browser-extension/blob/test/integration-tests/src/app/router/Welcome/Welcome.tsx/#L87

Reply:
yes and no. I see how it can be used but my branch doesn't even have this package `react-i18next`, looks like I'll need to merge all of these changes first.

Reply:
With a little bit of luck rebase should work ;)

Reply:
Is latest master already merged/rebased @pseudozach ? If there any issues I can support with that.

Reply:
this is added